test-suite/squidconf/tls-certs

```diff
@@ -0,0 +1,256 @@
+## Copyright (C) 1996-2018 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+#
+# The following tests check the sparse matrix of (valid) configs:
+#
+#  A) tls-cert PEM contents:
+#    1) cert only           tls-cert=./leaf-key-rsa.pem
+#    2) full chain          tls-cert=./leaf-chain-rsa.pem
+#    3) root CA             tls-cert=./ca-root-rsa.pem
+#    4) incomplete chain    tls-cert=./ca-mid-rsa.pem
+#    5) missing
```

Comment:
That connotation matches the situation. These tests are for checking what happens when the parameter is missing. Sometimes it can be omitted, but mostly required - ergo .. missing.

Reply:
> Sometimes it can be omitted

This fact means that we should avoid negative connotations. We should use a neutral one.

Reply:
Done.